services/crypto_get_info.proto

```diff
@@ -59,6 +59,64 @@ message CryptoGetInfoResponse {
      */
     ResponseHeader header = 1;
 
+    message CryptoAllowance {
+        /**
+         * The account ID of the hbar allowance spender.
+         */
+        AccountID spender = 1;
+
+        /**
+         * The current balance of the spender's allowance in tinybars.
+         */
+        int64 amount = 2;
+    }
+
+    message TokenAllowance {
+        /**
+         * The token that the allowance pertains to.
+         */
+        TokenID tokenId = 1;
+
+        /**
+         * The account ID of the token allowance spender.
+         */
+        AccountID spender = 2;
+
+        /**
+         * The current balance of the spender's token allowance.
+         */
+        int64 amount = 3;
+
+        /**
+         * The number of decimal places the token allowance value is divisible by.
+         */
+        int32 decimals = 4;
+    }
+
+    message NftAllowance {
```

Comment:
Same thing, can consolidate with `NftApproval`.

Reply:
Move to `NftAllowance` everywhere.